src/commands/run.js

```diff
@@ -1,17 +1,19 @@
 const { ApifyCommand } = require('../lib/apify_command');
 const execWithLog = require('../lib/exec');
-const { LOCAL_ENV_VARS, PROXY_PASSWORD_ENV_VAR } = require('../lib/consts');
+const { LOCAL_ENV_VARS, APIFY_PROXY_PASSWORD_ENV_VAR,
+    APIFY_USER_ID_ENV_VAR, APIFY_TOKEN_ENV_VAR } = require('../lib/consts');
 const { getLocalUserInfo } = require('../lib/utils');
 
 // TODO: Shall we also pass APIFY_TOKEN ?
 
 class RunCommand extends ApifyCommand {
     static async run() {
-        const { proxy } = getLocalUserInfo();
+        const { proxy, id: userId, token } = getLocalUserInfo();
         const env = Object.assign(process.env, LOCAL_ENV_VARS);
```

Comment:
Maybe would be better to use underscores `_.defaults(process.env, LOCAL_ENV_VARS)`. This we we allow user to override it.